documentation/book/security.adoc

```diff
@@ -1,61 +1,75 @@
+`<cluster-name>-cluster-ca`::
+Contains the private and public keys of the cluster CA wich is used for signing server certificates for the Kafka and Strimzi components (Kafka brokers, Zookeeper nodes and so on).
+`<cluster-name>-cluster-ca-cert`::
+Contains only the public key of the cluster CA.
+The public key should be used by Kafka clients to verify the identity of the Kafka brokers they are connecting to.
+`<cluster-name>-clients-ca`::
+Contains the private and public keys of the clients CA.
+The clients CA is used for TLS client authentication of Kafka clients when connecting to Kafka brokers.
+`<cluster-name>-clients-ca-cert`::
+Contains only the public key of the client CA.
+The public key is used by the Kafka brokers to verify then identity of Kafka clients when TLS client authentication is used.
+`<cluster-name>-kafka-brokers`::
+Contains all the Kafka broker private and public keys (certificates signed with the cluster CA).
+`<cluster-name>-zookeeper-nodes`::
+Contains all the Zookeeper node private and public keys (certificates signed with the cluster CA).
 
 All the keys are 2048 bits in size and are valid for 365 days from initial generation.
 
 NOTE: "certificates rotation" for generating new ones on their expiration will be supported in future releases.
 
-=== Listeners
+=== Kafka Listeners
 
 The data encryption is provided on two different listeners exposed by each Kafka broker.
 
-Interbroker communication is through the `REPLICATION` listener on port 9091, which is encrypted by default.
+Communication between brokers and between the different Kafka and Strimzi components is done through the `REPLICATION` listener on port 9091, which is encrypted by default.
 
 Encrypted communication with clients is provided through the `CLIENTTLS` listener on port 9093.
 
 NOTE: Un-encrypted communication with clients is still possible through the `CLIENT` listener on port 9092.
 
-=== Clients connection via TLS
+=== Kafka Client connections via TLS
 
-If a Kafka client wants to connect to the encrypted listener (CLIENTTLS) on port 9093, it needs to trust the client's CA certificate in order to verify the broker certificate received during the SSL/TLS handshake.
-The clients CA certificate can be extracted from the generated `<cluster-name>-kafka-cert` Secret with the following command if the Kafka cluster is running on {OpenShiftName}.
-
-[source,shell]
-oc get secret <cluster-name>-kafka-cert -o jsonpath='{.data.clients-ca\.crt}' | base64 -d > clients-ca.crt
+If a Kafka client wants to connect to the encrypted listener (CLIENTTLS) on port 9093, it needs to trust the cluster CA certificate in order to verify the broker certificate received during the SSL/TLS handshake.
+The cluster CA certificate can be extracted from the generated `<cluster-name>-cluster-ca-cert` Secret.
```

Comment:
Should "s" be a small letter in Secret?

Reply:
It is a name of an OpenShift resource. We usually use capital letter for those to distinguish it from just regular secret. I can change it though if you think it would be better that way.

Reply:
It's a resource, so it should be in monospace.